build.gradle.kts

```diff
@@ -3,6 +3,10 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 plugins {
     kotlin("jvm") version "1.3.21"
     groovy
+    maven
+    idea
+    `java-gradle-plugin`
+    id("com.palantir.idea-test-fix") version ("0.1.0")
```

Comment:
I assume it's not possible to unify way of providing plugins, is it?

Reply:
it can be done with `apply(plugin = "some-plugin")` but this way is recommended in gradle docs